alinka/widget/containers/main_body/content/application/application_tabs/meeting_tab/__init__.py

```diff
@@ -30,6 +31,9 @@ def __init__(self, parent: QWidget):
         self.model = QStandardItemModel()
         self.model.itemChanged.connect(self.item_changed)
         self.listView = QListView(self.meeting_member_group)
+
+        self.populate_meeting_members()
```

Comment:
why do we need to run it here? I recall I moved that to `showEvent` (as otherwise new members from settings would not be visible without restaring application)